app/routes.py

```diff
@@ -1,2 +1,52 @@
-from flask import Blueprint
+from app import db
+from app.models.planet import Planet
+from flask import Blueprint, jsonify, make_response, request
 
+planets_bp = Blueprint("planets", __name__, url_prefix="/planets")
+
+@planets_bp.route("", methods=["GET", "POST"])
+def handle_planets():
+    if request.method == "GET":
+        planets = Planet.query.all()
+        planets_response = []
+        for planet in planets:
+            planets_response.append(
+                planet.to_json()
+            )
+        return jsonify(planets_response)
+    
+    elif request.method == "POST":
+        request_body = request.get_json()
+        new_planet = Planet(name=request_body["name"],
```

Comment:
Consider how you could handle an incomplete request_body that doesn't have `name` or `description`